app/frontend/src/registry/components/people/OrganizationAdd.vue

```diff
@@ -153,7 +145,7 @@
           <b-row class="my-3">
             <b-col>
               <b-button type="submit" class="mr-2" variant="primary" :disabled="orgSubmitLoading">Save</b-button>
-              <b-button type="reset" variant="light" id="orgFormResetButton">Cancel</b-button>
```

Comment:
this id is used by unit tests (specifically, to check that the form object data is cleared when this button is clicked). Any way we can keep it in?

There are a number of unit tests that basically say "click button with id #formSubmit, assert that the handleSubmit function ran"